conf/defaults.ini

```diff
@@ -1263,6 +1263,9 @@ default_baselayer_config =
 # Enable or disable loading other base map layers
 enable_custom_baselayers = true
 
+# Correlations page
+correlations = false
```

Comment:
i don't think this is required fyi. feature toggles are disabled by default 😌

Reply:
though actually i am a bit confused - this config option is under `geomap`?

Reply:
Removed in 0431ff1ab051d0d7f0f68a407e57c465ee910d4b 👍